torch/csrc/jit/mobile/import.cpp

```diff
@@ -84,6 +84,7 @@ void print_unsupported_ops_and_throw(
 
 void parseMethods(
     const std::vector<IValue>& vals,
+    const std::vector<IValue>& debug_info_vals,
```

Comment:
Maybe do optional here, so that instead of `bool has_debug_info = debug_info_vals.size() > 0;`, you can do `bool has_debug_info = debug_info_vals.has_value();`